tests/conftest.py

```diff
@@ -1,14 +1,29 @@
-import gc
-import os
 import pytest
-import pyvista
 from pyvista.plotting import system_supports_plotting
 
 NO_PLOTTING = not system_supports_plotting()
 
+
+def _check_qt_installed():
+    try:
+        from qtpy import QtCore  # noqa
+    except ModuleNotFoundError:
+        return False
+    else:
+        return True
+
+
 @pytest.fixture()
 def plotting():
     """Require plotting."""
     if NO_PLOTTING:
         pytest.skip(NO_PLOTTING, reason="Requires system to support plotting")
     yield
+
+
+@pytest.fixture()
+def no_qt():
+    """Require plotting."""
+    if _check_qt_installed():
+        pytest.skip(reason="Requires that Qt is not installed")
```

Comment:
instead of doing this, I would `monkeypatch.setattr` something to make it behave like nothing is installed. It might be as easy as `monkeypatch.setenv('QT_API', 'bad_name')` or so, since it does appear to cause a failure loally at least:
```
$ QT_API=bad_name python -c "from qtpy import QtCore"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/larsoner/.local/lib/python3.10/site-packages/qtpy/__init__.py", line 106, in <module>
    assert API in (PYQT5_API + PYQT6_API + PYSIDE2_API + PYSIDE6_API)
AssertionError
```

Reply:
But it would be a different test, right? Compared to really checking the installation. Plus I think `qtpy` throws a specific `PythonQtError` when no Qt API is found, the one you mention is an failed assertion.

Reply:
... my suggested change makes it so that regardless of whether or not the dev's system has Qt installed, the test runs and checks what it's supposed to check. What you have now requires that the system *not* have Qt in order to run, which will not be the case on almost any dev's system, rendering the test a huge pain to actually execute locally.

Reply:
I'll work on this, like you said, maybe there is a way to make `qtpy` behave like nothing is installed :+1:

Reply:
I think you just put a `monkeypatch.setenv('QT_API', 'bad_name')` inside the conditional here instead of a `pytest.skip` and everything should just work...

Reply:
But seeting `QT_API` is like requesting the wrong API :sweat_smile: 

I think I found a way by looking at `qtpy` code. It checks the internal boolean variables `PYQT6`, `PYQT5`, `PYSIDE2`, `PYSIDE6` and if none is true, it raises `PythonQtError`. It seems close enough to no Qt API found, WDYT?

Reply:
> But setting QT_API is like requesting the wrong API ￼

That is exactly the point, to make it (temporarily, via monkeypatch) fail on systems where it would otherwise pass. This is what I'm trying to accomplish...

> It checks the internal boolean variables PYQT6, PYQT5, PYSIDE2, PYSIDE6 and if none is true, it raises PythonQtError. It seems close enough to no Qt API found, WDYT?

Maybe but it seems more fragile to mess with `qtpy` internals rather than relying on a public API (QT_API) that we know we can use to make Qt appear to be missing.

I still don't understand the hesitation to route via QT_API?

Reply:
> I still don't understand the hesitation to route via QT_API?

Sorry if I feel picky, I just want to test a scenario as close as possible to the real situation. But it's not necessary really, as long as we keep the CI job that tests without any Qt backend installed, I'll be happy :+1:

Reply:
If you think you can do it easily/cleanly/as easily maintained via a `monkeypatch` directly to `qtpy` (or some other method) feel free. I really just want to make sure that we can emulate `qtpy` failing to import a Qt API / raising some error even on systems where it would normally succeed so that the `no_qt` tests can actually run for devs